SWLOR.Game.Server/Service/AbilityService/RecastGroup.cs

```diff
@@ -177,6 +177,12 @@ public enum RecastGroup
         Rejuvenation = 84,
         [RecastGroup("Frenzied Shout", "Frenz. Shout")]
         FrenziedShout = 85,
+        [RecastGroup("Force Death", "Force Death")]
+        ForceDeath = 85,     
+        [RecastGroup("Tutaminis", "Tutaminis")]
+        Tutaminis = 86,
+        [RecastGroup("Thermal Detonator", "Thermal Detonator")]
```

Comment:
There's a limit of about 14 characters (depending on character used) for certain UI elements. Please shorten the second string to below this limit. I would recommend "T. Detonator"